src/operator/rnn_impl.h

```diff
@@ -174,17 +174,21 @@ void LstmForwardTraining(DType* ws,
       w_ptr += w_size;
       b_ptr += b_size;
       if (dropout > 0.0f) {
-        #pragma omp parallel for num_threads(omp_threads)
-        for (int j = 0; j < T * N * H * D; j++) {
-          int rand_data = rand_r(&seed_);
-          if (static_cast<float>(rand_data % 1000) < static_cast<float>(1000 * dropout)) {
-            dropout_random[i * T * N * H * D + j] = 0;
-            y.dptr_[j] = 0;
-          } else {
-            dropout_random[i * T * N * H * D + j] = 1.0f - dropout;
-            y.dptr_[j] =  y.dptr_[j] / (1.0f - dropout);
+          #pragma omp parallel for num_threads(omp_threads)
+          for (int j = 0; j < T * N * H * D; j++) {
```

Comment:
size_t?

Reply:
Maybe I'll leave it for later improvements, since it's not related to the main change